Tools/clinic/clinic.py

```diff
@@ -1875,7 +1875,7 @@ def print_block(self, block, *, core_includes=False):
                 output += '\n'
             write(output)
 
-        arguments="output={} input={}".format(compute_checksum(output, 16), compute_checksum(input, 16))
+        arguments=f"output={compute_checksum(output, 16)} input={compute_checksum(input, 16)}"
```

Comment:
In my eyes, the f-string is still too cluttered. How about:

```suggestion
        arguments="output={output}, input={input}".format(
            output=compute_checksum(output, 16),
            input=compute_checksum(input, 16)
        )
```

Reply:
I prefer the f-string myself, but sure!